lib/src/main/java/growthbook/sdk/java/multiusermode/configurations/UserContext.java

```diff
@@ -0,0 +1,53 @@
+package growthbook.sdk.java.multiusermode.configurations;
+
+import com.google.gson.JsonObject;
+import growthbook.sdk.java.multiusermode.util.TransformationUtil;
+import growthbook.sdk.java.stickyBucketing.StickyAssignmentsDocument;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.annotation.Nullable;
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+@Builder
+@Slf4j
+public class UserContext {
+
+    @Nullable
+    private JsonObject attributes;
+
+    @Nullable
+    private String url;
+
+    @Nullable
+    private Map<String, StickyAssignmentsDocument> stickyBucketAssignmentDocs;
+
+    @Nullable
+    private Map<String, Integer> forcedVariationsMap;
+
+    @Nullable
+    private Map<String, Object> forcedFeatureValues;
+
+    @Nullable
+    private String attributesJson;
+
+    public JsonObject getAttributes() {
+        if (this.attributes == null) {
+            return new JsonObject();
+        }
+
+        return this.attributes;
+    }
+
+    public Map<String, Integer> getForcedVariationsMap() {
+        return this.forcedVariationsMap == null ? new HashMap<>() : this.forcedVariationsMap;
+    }
```

Comment:
Most of the time, `forcedVariationsMap` is going to be null.  This will create a new `HashMap` every time an experiment is evaluated.  For performance, should we do this once instead and re-use the object?  Maybe creating an empty HashMap is super lightweight and it doesn't matter, so feel free to ignore if that's the case.

Reply:
creating an empty hashmap is far more simple and super lighweight. it would be gc'ed as soon as the object holding it dies.